app/src/main/java/com/eveningoutpost/dexdrip/Models/Calibration.java

```diff
@@ -875,7 +877,7 @@ private double calculateWeight() {
     }
 
 
-    public static void adjustRecentBgReadings(int adjustCount) {
+    public static void adjustRecentBgReadings(int adjustCount, SharedPreferences prefs) {
```

Comment:
I haven't really examined any other part of this PR, but don't adjust the method signature to pass in an instance of `SharedPreference`s, just use the `Pref` class static methods instead.

Reply:
Solid tip Jam, thanks. I've made that change.

Feel free to hold off fully reviewing this, I had left it as a draft PR so I could provide some additional info - I'm guessing the creation of PR notified you, where I assumed it would be on submission!